lib/ts/recipe/session/accessToken.ts

```diff
@@ -31,21 +32,44 @@ export async function getInfoFromAccessToken(
     timeCreated: number;
 }> {
     try {
-        verifyJWT(jwtInfo, jwtSigningPublicKey);
-        const payload = jwtInfo.payload;
+        // From the library examples
+        const { payload } = await jose.jwtVerify(jwtInfo.rawTokenString, jwks).catch(async (error) => {
```

Comment:
Is the `.catch` async? Cause then you should convert this to promise style and await and then catch the error, otherwise `validateAccessTokenStructure` in the next line will start running